wavefront-adapter-for-istio/templates/handler.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   params:
     {{- if and (.Values.adapter.useProxy)  (.Values.proxy.enabled) }} 
     proxy:
-      address: {{ .Release.Name }}-wavefront-proxy.{{ .Release.Namespace }}.svc.cluster.local:2878
+      address: {{ .Release.Name }}-proxy.{{ .Release.Namespace }}.svc.cluster.local:2878
```

Comment:
Wonder why this differs from the prometheus storage adapter:

https://github.com/wavefrontHQ/helm/blob/961d8f9037818aa9460f755012b77eac934a8689/prometheus-storage-adapter/templates/adapter-deployment.yaml#L31

I think it worked fine for that?

Reply:
This service is getting created `wavefront-istio   wavefront-adapter-for-istio-proxy` when wavefront proxy is installed. Not sure why it is different from storage adapter.